uiowa/uids
# Feature: Sprint 14 work for sitenow integration and minor uids updates

# How to test

#125 
- Verify that `@include flex($fg: 1, $fs: 1, $fb: 0%);` has been removed from card-block.scss
- Verify that card has `background: #fff;` set. 

#122 
- Verify that new danger color has updated dark orange: http://localhost:3000/components/detail/alerts--danger

#106 
- Go to http://localhost:3000/components/detail/alerts--system-message and verify that there is a generic system message variant. 

#121 
- Go to http://localhost:3000/components/detail/banner-block and change class to `banner banner--reversed` and verify that black gradient is darker and consistent. 

#127 
- Go to http://localhost:3000/components/detail/card-block--default and verify that card__author field is slightly larger at `1.1rem`
